flex-config/ui_attributes.common.json

```diff
@@ -224,7 +224,10 @@
             "require_disposition": true,
             "dispositions": ["Promotional Sale", "Renewal"]
           }
-        }
+        },
+        "text_attributes": [],
+        "select_attributes": [],
+        "multi_select_group" : {}
```

Comment:
this config should probably be supported by queue like the disposition config.  

I think we also want to change the "queue" configuration to support queue name as well as queue sid - i done something similar for [activity skill filter](https://github.com/twilio-professional-services/flex-project-template/pull/312/files) that might be worth taking a look at. Could i ask you make the tweak to match config on queue name or queue sid since you are in this space?

Reply:
I think we may be over-engineering this and adding unnecessary levels of complexity that most customers probably won't need.  Typically a wrap-up form contains a set of generic questions that apply to all types of calls/contacts. Any Text and Select fields on this wrap-up form can already be marked as NOT required so it's flexible enough to support different use cases where not all options may not be applicable for each call/queue.  For example, a list of Topics (Reasons for the Call) should be a comprehensive superset of all possible reasons customer could be calling/contacting a business.  We should try to balance Perfection vs Good Enough. I think what we provide here in the Extended Wrap up form with customizable Text Boxes and Pick from a list (Select/Pulldown menu) should cover the vast majority of use cases.

Reply:
Done.  You can now also configure all attribute types (Text, Select, Checkboxes) per queue.  Updated to code to match by queue name.